test/psse.jl

```diff
@@ -173,4 +170,23 @@ end
         setlevel!(getlogger(PowerModels), "error")
     end
 
+    @testset "three-winding transformer" begin
+        data_pti = PowerModels.parse_file("../test/data/pti/three_winding_test.raw")
+        data_mp = PowerModels.parse_file("../test/data/matpower/three_winding_test_mp.m")  # Matpower Translation of PTI file
+        data_pw = PowerModels.parse_file("../test/data/matpower/three_winding_test_pw.m")  # PowerWorld Translation of PTI file
+
+        set_costs!(data_mp)
+        set_costs!(data_pw)
+
+        result_pti = PowerModels.run_opf(data_pti, PowerModels.ACPPowerModel, ipopt_solver)
+        result_mp = PowerModels.run_opf(data_mp, PowerModels.ACPPowerModel, ipopt_solver)
+        result_pw = PowerModels.run_opf(data_pw, PowerModels.ACPPowerModel, ipopt_solver)
+
+        @test result_pti["status"] == :LocalOptimal
```

Comment:
In addition to solving an OPF, I would also do some checks on the data it self.  For example, the number of buses and transformers in the PowerModels data model.